pkgs/data/fonts/kochi-substitute-naga10/default.nix

```diff
@@ -5,7 +5,7 @@ in
 fetchzip {
   name = "kochi-substitute-naga10-${version}";
 
-  url = "mirror://sourceforgejp/efont/5411/kochi-substitute-${version}.tar.bz2";
+  url = "mirror://osdn/efont/efont/5411/kochi-substitute-${version}.tar.bz2";
```

Comment:
This should only have one `efont/` right?

Reply:
yes, fixed. Thanks!